google/cloud/spanner/database_admin_connection.h

```diff
@@ -174,11 +175,11 @@ class DatabaseAdminConnection {
     /// The name of the database.
     Database database;
     std::string backup_id;
-    std::chrono::system_clock::time_point expire_time;
+    Timestamp expire_time;
```

Comment:
Terrible idea ahead: If `Timestamp` were _implicitly_ constructible from `std::chrono::system_clock::time_point`, then existing code like
```cc
std::chrono::system_clock::time_point tp;
auto p = CreateBackupParams{db, id, tp, {}};
```
Would continue to compile even if we changed the `expire_time` field to `Timestamp`. It's probably still a breaking change technically, but perhaps less so.

Reply:
We've taken the "leave the old fields" approach, so this is moot.  The possibility of the conversion failing would have been problematic anyway.